### ISSUE TITLE
Nodejs app crash when running inside node:18.17-alpine3.18 container

### ISSUE THREAD

**Issue by OferLazmi** [duckdb/duckdb-node#43]
same app in running as expected in windows machine but when deploying it into node:18.17-alpine3.18 container it crashes
i removed the duckDb code and everything works.

Here is the code that causes the crash:

return new Promise(async (resolve, reject) => {
            try {
                let query = `                    
                    INSTALL httpfs;
                    LOAD httpfs;                    
                `;
                this.db = new duckdb.Database(':memory:');
                this.db.all(query, (err, res) => {
                    if (err) {             
                        this.logger.error(`${this.installExtensions.name} got unhandled errors:`, err);           
                        reject(err);
                        return;
                    }
                    resolve(res);
                });
            } catch (error) {
                this.logger.error(`${this.installExtensions.name} got unhandled error:1`, error);
                reject(error);
            }
        });
        
did someone encountered this kind of problem?

**Comment by nabriski** [comment]
yes, I had to switch to a Debian image. It appears the duckdb is not compiled to run on Alpine.

**Comment by rguo123** [comment]
any update here? running into same issue for node:20-alpine. Node app crashes on `await Database.create(':memory:');`

Tried installing dependencies listed in other issues but none of these helped

```
RUN apk add --update gcc g++ make python3 clang cmake ninja git linux-headers
RUN apk add --update libc6-compat gcompat
```
